### PR TITLE
Adding test case for handling pipe

### PR DESCRIPTION
**Pull request by VigneshMurugan**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by kibertoad** [comment]
wouldn't using async/await simplify code a bit?

File: test/listen.test.js [hunk 1]
```diff
@@ -351,3 +351,23 @@ test('listen logs the port as info', t => {
       t.ok(/http:\/\//.test(msgs[0]))
     })
 })
+
+test('listen when firstArg is string(pipe) and without backlog', t => {
+  t.plan(1)
+  const fastify = Fastify()
+  t.tearDown(fastify.close.bind(fastify))
+  fastify.listen('\\\\.\\pipe\\testPipe')
```
Comment (mcollina) [inline test/listen.test.js:359]: can you please convert this to `await fastify.listen('\\\\.\\pipe\\testPipe')`

**Review by mcollina — CHANGES_REQUESTED** [review]

File: test/listen.test.js (hunk 1 above)
Comment (VigneshMurugan) [reply test/listen.test.js]: Completed. Thanks for the suggestion !

**Review by mcollina — APPROVED** [review]
lgtm, great!

**Comment by VigneshMurugan** [comment]
@mcollina Can you please merge this PR ?

**Review by kibertoad — APPROVED** [review]

**Pull request merged**